docs/sources/set-up/install/puppet.md

```diff
@@ -27,7 +24,7 @@ You can use Puppet to install and manage {{< param "PRODUCT_NAME" >}}.
 
 To add {{< param "PRODUCT_NAME" >}} to a host:
 
-1. Ensure that the following module dependencies are declared and installed:
+1. Make sure that the following module dependencies are declared and installed:
 
     ```json
     {
```

Comment:
below is a ref to grafana_agent: `class grafana_agent::grafana_agent_flow ()`

Reply:
I missed it :-(  Fixed in a new commit.